src/System.Security.Permissions/src/System.Security.Permissions.csproj

```diff
@@ -171,6 +181,9 @@
     <Compile Include="System\ServiceProcess\ServiceControllerPermissionEntryCollection.cs" />
     <Compile Include="System\Transactions\DistributedTransactionPermission.cs" />
     <Compile Include="System\Transactions\DistributedTransactionPermissionAttribute.cs" />
+    <Compile Include="System\Web\AspNetHostingPermission.cs" />
+    <Compile Include="System\Web\AspNetHostingPermissionAttribute.cs" />
+    <Compile Include="System\Web\AspNetHostingPermissionLevel.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsNetFx)' == 'true'">
     <Reference Include="mscorlib" />
```

Comment:
My guess is that you are missing some entries here -- System.Configuration, System.Core, System.Security, System.DirectoryServices

Reply:
Add those and push to the PR, while that's running you can also build the netfx configuration locally